StantUiIosLibraryDemo/StantUiIosLibraryDemoTests/SyncDialogTests.swift

```diff
@@ -0,0 +1,166 @@
+//
```

Comment:
Tô achando esse teste um pouco inchado, teria como destrinchar o corpo das funções pra não misturar asserção/setup? Até mesmo em outros arquivos, se for preciso

Reply:
Dei uma refatorada, no mesmo arquivo.. Mas dividi melhor, vê se tá bom :)